site/gatsby-site/src/scripts/translateReports.js

```diff
@@ -0,0 +1,62 @@
+require('dotenv').config();
+
+const config = require('../../config');
+
+const { MongoClient } = require('mongodb');
+
+const { Translate } = require('@google-cloud/translate').v2;
+
+const Translator = require('./Translator');
+
+const { getLanguages } = require('../../i18n');
+
+const reporter = { log: console.log, error: console.error, warn: console.warn };
+
+(async () => {
+  let mongoClient;
+
+  try {
```

Comment:
nitpick: move the log above the try. Let's keep `try` as close as possible to failing logic.